keepassxc-browser/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
 	"manifest_version": 2,
 	"name": "KeePassXC-Browser",
-	"version": "0.5.0",
+	"version": "0.5.2",
```

Comment:
From here on, we shouldn't increase the version anymore for every minor change.

Reply:
Yes we shouldn't. But it's strange because didn't that already changed in https://github.com/keepassxreboot/keepassxc-browser/pull/8 ?

Reply:
Yeah, weird.